Invoice_to_Payment_Process/Invoice_to_Payment_Process.md

```diff
@@ -109,6 +109,51 @@ graph LR
 
 ### 3.1 Message Level Response
 
+#### 3.1.1 Scope
+
+The scope of the Message Level Response is:
+
+1. The Message Level Response uses BIS Message Level Response 3.0 (see this link for the [Peppol business documentation|https://docs.peppol.eu/poacc/upgrade-3/profiles/36-mlr/]. The technical UBL message used is ApplicationResponse. See paragraph 5 for the message definition.
+2. The Message Level Response is sent by the customer AP (C3) on behalf of the customer (C4) to the supplier (C1) but can be handled by the supplier AP (C2) conform standard Peppol exchange mechanisms (including use of SML and SMP) and always refers to the envelop of the business document sent by the supplier.
+3. The SMP record for the MLR should use the sending AP as endpoint or route the message to the supplier AP (C2).
+4. The following errors are within the scope for a negative/rejecting Message Level Response:
+ 	* XML schema validation error
+	* Standard Compliance violations (e.g. empty elements not being allowed by UBL 2.1)
+	* Validation error of type fatal error
+	* Validation error of type warning. Warnings alone must NOT cause rejection of the business document (but they may be reported in addition to fatal errors)
+	* Wrong version of business document (Will be handled like validation error of type fatal error)
+5. Can be used to indicate delivery failures between C3 and C4 when the MLR is mandatory and there is no reject or accept send by MLR within a specific time (1 hour)
```

Comment:
Not according to current discussions within OpenPeppol. They suggest that a delivery failure is indicated by a missing Invoice Response within 1 hour.

Reply:
Added that this is a sugestion and has a dependency